app/login/styles.css

```diff
@@ -0,0 +1,79 @@
+/* every page */
+
```

Comment:
I've already removed some of the CSS that was not needed by this page but this top section could be shared. Is this something that is better done by SASS?

Reply:
Yes, absolutely.

There are lots of potential options for making the CSS less torturous, but honestly SASS is just the best way to do it.